probe-rs-cli-util/src/common_options.rs

```diff
@@ -237,13 +237,16 @@ impl ProbeOptions {
             }
         }?;
 
-        // Select protocol and speed
-        probe.select_protocol(self.protocol).map_err(|error| {
-            OperationError::FailedToSelectProtocol {
-                source: error,
-                protocol: self.protocol,
-            }
-        })?;
+        if let Some(protocol) = self.protocol {
+            // Select protocol and speed
+            probe.select_protocol(protocol).map_err(|error| {
+                OperationError::FailedToSelectProtocol {
+                    source: error,
+                    protocol,
+                }
+            })?;
+        }
+
```

Comment:
Is JTAG the default protocol now?

Reply:
It depends on the probe. Some probes have SWD as default, some JTAG.

probe-rs could probably be a bit smarter and automatically select JTAG for RISCV, but that will also not work if we try to select SWD first.